src/classes/Passage.ts

```diff
@@ -164,20 +170,25 @@ export default class Passage {
      * Create a Magic Link for your app.
      *
      * @param {MagicLinkRequest} magicLinkReq options for creating a MagicLink.
-     * @return {Promise<MagicLinkObject>} Passage MagicLink object
+     * @return {Promise<MagicLink>} Passage MagicLink object
      */
-    async createMagicLink(magicLinkReq: MagicLinkRequest): Promise<MagicLinkObject> {
+    async createMagicLink(magicLinkReq: CreateMagicLinkRequest): Promise<MagicLink> {
         try {
-            const response = await fetch({
-                body: magicLinkReq,
+            const configuration = new Configuration({
```

Comment:
@vanessa-passage 

Can we make a singleton client for every method that needs an API key (all but one I think) and then only use this pattern in getApp or anywhere else we don't need the apikey?